comm_config.yaml.tpl

```
@@ -15,6 +15,8 @@ communications:
             sources:
               - 'k8s-events'
       token: 'SLACK_API_TOKEN'
+      botToken: 'SLACK_BOT_TOKEN'
+      appToken: 'SLACK_APP_TOKEN'
```

please also reset the values here:

```suggestion
      token: "" # SLACK_API_TOKEN
      botToken: "" # SLACK_BOT_TOKEN
      appToken: "" # SLACK_APP_TOKEN
```